packages/zmarkdown/config/latex/prepare-quizz.js

```diff
@@ -0,0 +1,22 @@
+const visit = require('unist-util-visit')
+
+module.exports = processQuizzFactory
+const clone = require('clone')
+function processQuizzFactory (ctx) {
+  const correctionTitle = ctx.correctionTitle || 'Correction'
+  return function quizzCustomBlockVisitor (node, index, parent) {
+    node.type = 'neutralCustomBlock'
+    const correction = clone(node)
+    correction.type = 'sCustomBlock'
+    correction.children[0].children[0].value = correctionTitle
```

Comment:
Please check that `correction.children[0].children[0]` exists and do something if it doesn't, we had a lot of problems with direct indexing like this in the past. Maybe simply revert the change on `node.type` and return?

Reply:
logically we ensure that quizz has a title so the children[0].children[0] MUST be there.

Reply:
Well, I may be a bit too paranoid here; my point was: it cost nothing to add a test and we already had problems with theses kind of elements that "we thought would be here but are not" :D
Maybe we could skip the tests.